src/types/case-insensitive-text.ts

```diff
@@ -0,0 +1,13 @@
+import * as _Promise from 'bluebird';
```

Comment:
Why is this import needed?

Reply:
Because we generate declaration files and `validate` returns a bluebird promise, so we need this unused import to have the bluebird types available for the declaration file, it's weird but it breaks without it